src/components/Burger/Burger.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {render} from '@testing-library/react';
+import Burger from './index';
+
+describe('Test for Burger Component', () => {
+    test('Review that component contains a button', () => {
```

Comment:
If I read the description as "test the Burger Component contains a button", that would make more sense to me.

Thus, in the code it would be:
```suggestion
    test('the Burger Component contains a button', () => {
```